demos/imagenet_classification_demo/imagenet_classification_demo.hpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+
+static const char help_message[] = "Print a usage message.";
+static const char image_message[] = "Required. Path to a folder with images or path to an image files: a .ubyte " \
+                                    "file for LeNet and a .bmp file for the other networks.";
+static const char model_message[] = "Required. Path to an .xml file with a trained model.";
+static const char classes_message[] = "Required. Path to .txt file with image classes.";
+static const char labels_message[] = "Required. Path to .txt file with imagenet labels.";
+static const char target_device_message[] = "Optional. Specify the target device to infer on (the list of available " \
+                                            "devices is shown below). Default value is CPU. " \
+                                            "Sample will look for a suitable plugin for device specified.";
+static const char batch_size_message[] = "Optional. Specify batch to infer. " \
+                                         "Default value is 1.";
+static const char num_threads_message[] = "Optional. Specify count of threads.";
+static const char num_streams_message[] = "Optional. Specify count of streams.";
+static const char num_inf_req_message[] = "Optional. Number of infer requests.";
+static const char delay_message[] = "Optional. Delay between screen updates in milliseconds. " \
+                                    "Default value is 1.";
+static const char image_grid_resolution_message[] = "Optional. Set image grid resolution in format WxH. " \
+                                                    "Default value is 1920x1080.";
+static const char ntop_message[] = "Optional. Number of top results. Default value is 10.";
+static const char custom_cldnn_message[] = "Required for GPU custom kernels. " \
+                                           "Absolute path to the .xml file with kernels description.";
+static const char custom_cpu_library_message[] = "Required for CPU custom layers." \
+                                                 "Absolute path to a shared library with the kernels implementation.";
+static const char no_show_message[] = "Optional. Disable showing of processed images.";
+static const char execution_time_message[] = "Optional. Time in seconds to execute program. " \
+                                             "Default is -1 (infinite time).";
+static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+
+DEFINE_bool(h, false, help_message);
+DEFINE_string(i, "", image_message);
+DEFINE_string(m, "", model_message);
+DEFINE_string(classes, "", classes_message);
+DEFINE_string(labels, "", labels_message);
+DEFINE_string(d, "CPU", target_device_message);
+DEFINE_uint32(b, 1, batch_size_message);
+DEFINE_uint32(nthreads, 0, num_threads_message);
+DEFINE_string(nstreams, "", num_streams_message);
+DEFINE_uint32(nireq, 0, num_inf_req_message);
+DEFINE_uint32(nt, 10, ntop_message);
```

Comment:
The default value should be 1 or 5 because usually papers report top1 and top5 accuracy.